jib-build-plan/src/main/java/com/google/cloud/tools/jib/api/buildplan/FileEntriesLayer.java

```diff
@@ -89,7 +89,9 @@ public Builder addEntry(FileEntry entry) {
      * @param pathInContainer the path in the container file system corresponding to the {@code
      *     sourceFile}
      * @return this
+     * @deprecated Use {@link #newAddEntry(Path, AbsoluteUnixPath)}
      */
+    @Deprecated
```

Comment:
I'm not sure we have to deprecate everything in here. If the new methods do the same thing, there is no reason to make the new methods.

We might only want to really change the methods that expose the bifunction in the api?